Main/ClientApp/src/Components/BaseCollectionView.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactNode } from "react";
+import { Collection } from "../Model/Collection";
+import ElementCreator from "./ElementCreator";
+import { GridList, GridListTile, Fab } from "@material-ui/core";
+import { Theme, createStyles, makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    list: {
+      padding: "25px 30px 25px 30px"
+    }
+  })
+);
+
+type BaseCollectionViewProps = {
+  collection: Collection | null | undefined;
```

Comment:
do we really need both undefined AND null? Please avoid this as much as possible